tf-modules/cumulus-rds-tf/outputs.tf

```diff
@@ -13,7 +13,7 @@ output "admin_db_login_secret_arn" {
 output "admin_db_login_secret_version" {
   value = aws_secretsmanager_secret_version.rds_login.version_id
 }
-
+# Return aws_secretsmanager_secret.rds_login.arn in case var.provision_user_database is false
 output "user_credentials_secret_arn" {
-  value = var.provision_user_database ? module.provision_database[0].database_credentials_secret_arn : null
+  value = var.provision_user_database ? module.provision_database[0].database_credentials_secret_arn : aws_secretsmanager_secret.rds_login.arn
```

Comment:
Can you explain a bit about why you want the output from this module to output the admin PW by default?   

It's not necessarily a generally a good practice to utilize the admin login, but perhaps more importantly you have the secret from `admin_db_login_secret_arn` already in the output, and combining these outputs seems to conflate those two differently-scoped outputs.

Reply:
Just in case the provisioner chose to have provision_user_database=false (the default value), they will need to have a secret ARN to be used somewhere else (data_migration1 and cumulus module)

Reply:
So if provision_user_database is set to false, the thought was this module would then create the cluster, and provisioning a user login/secret/etc would be at the module user's discretion.    

Given the nature of the module, I'm a bit concerned maintenance-wise about conflating those two things given the use case seems output/use specific and the logic to do so could be maintained in a client module that uses these outputs.